### ISSUE TITLE
still used?

### ISSUE THREAD

**Issue by tkimnguyen** [plone/plone.github.com#1]
Is http://plone.github.io still used @aclark4life @witsch ?

**Comment by aclark4life** [comment]
No, feel free to re-purpose or remove (as far as I'm concerned).

**Comment by davisagli** [comment]
Do the mockup docs exist somewhere else?

**Comment by witsch** [comment]
I have no idea… :)

**Comment by tkimnguyen** [comment]
@davisagli you mean http://plone.github.io/mockup/dev/#pattern/relateditems ?

**Comment by tkimnguyen** [comment]
I just didn't understand if "OK" was meant to tell any service that GitHub was up... :)

**Comment by davisagli** [comment]
Right, I guess the mockup stuff comes from a different repository

**Comment by aclark4life** [comment]
@tkimnguyen I think the original intention was to manually update plone.org & related status … but … yeah.

**Comment by tkimnguyen** [comment]
Would be nice to use something like https://www.statuspage.io if they'd support free software projects (Linode uses them)
